fn main() {
    panic!("Something went wrong!");
}
